assets/tests/components/placesPage.test.tsx

```diff
@@ -11,27 +11,11 @@ beforeAll(() => {
   document.body.appendChild(app);
 });
 
-describe("Dashboard", () => {
-  let originalFetch: any;
-
-  beforeEach(() => {
-    originalFetch = global.fetch;
-    global.fetch = jest.fn(() =>
-      Promise.resolve({
-        json: () => Promise.resolve(placesAndScreens),
-      })
-    ) as jest.Mock;
-  });
-
-  afterEach(() => {
-    global.fetch = originalFetch;
-  });
-
```

Comment:
So I did remove this fetching logic since now the Places array is passed as a prop to the component that this file is for. I'm not sure if that move makes sense so let me know if we should keep it.

Reply:
I'm not sure, but I'm fine with this. The when and where will the fetch be removed (I think it's still there in master)

Reply:
So, I _think_ the mock fetch needed to be there because it was called each time the dashboard component was rendered and that's how the places list was populated. Now that the separate Places and Alerts pages are basically just viewer components that take in their respective lists of things as props, the fetch doesn't run each time its re-rendered.

Now the Dashboard component is a lot thinner and doesn't do anything besides calling the backend or API for data, which we can't really unit test. So until there's some unit-testable logic that we add into the Dashboard component in a subsequent PR, I want to say that we can leave this mock fetching stuff out.